sections/shared/components/GasPriceSelect/GasPriceSelect.tsx

```diff
@@ -29,16 +23,20 @@ const GasPriceSelect: FC<GasPriceSelectProps> = ({ gasPrices, transactionFee, ..
 	const { t } = useTranslation();
 	const [gasSpeed] = useRecoilState<keyof GasPrices>(gasSpeedState);
 	const [customGasPrice] = useRecoilState(customGasPriceState);
-	const { selectedPriceCurrency } = useSelectedPriceCurrency();
 	const isMainnet = useRecoilValue(isMainnetState);
+	const isL2 = useRecoilValue(isL2State);
 
 	const hasCustomGasPrice = customGasPrice !== '';
 	const gasPrice = gasPrices ? parseGasPriceObject(gasPrices[gasSpeed]) : null;
 
-	const gasPriceItem = hasCustomGasPrice ? (
-		<span data-testid="gas-price">{formatNumber(customGasPrice, { minDecimals: 4 })}</span>
-	) : (
-		<span data-testid="gas-price">{formatNumber(gasPrice ?? 0, { minDecimals: 4 })}</span>
+	const gasPriceItem = (
+		<span data-testid="gas-price">
+			{isL2
+				? formatCurrency(Synths.sUSD, transactionFee ?? 0, { sign: '$', maxDecimals: 1 })
```

Comment:
One nitpick, I would prefer this doesn't show $0.00 when the input isn't valid. The other inputs show `-` when the inputs are not complete.